<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>591</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ICQ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/icon/res/ico.ico</normaloff>:/images/icon/res/ico.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="onlineComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>在线</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>隐身</string>
     </property>
    </item>
   </widget>
   <widget class="QTabWidget" name="commTabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>251</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="friendsTab">
     <attribute name="title">
      <string>好友</string>
     </attribute>
     <widget class="QTreeWidget" name="friendsTreeWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>252</width>
        <height>421</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>好友列表</string>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </column>
      <item>
       <property name="text">
        <string>我的好友</string>
       </property>
       <item>
        <property name="text">
         <string>新建子项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>朋友</string>
       </property>
       <item>
        <property name="text">
         <string>新建子项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>同学</string>
       </property>
       <item>
        <property name="text">
         <string>新建子项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
      </item>
     </widget>
     <widget class="QPushButton" name="findUserButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>424</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>找人</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="groupTab">
     <attribute name="title">
      <string>群</string>
     </attribute>
     <widget class="QListWidget" name="groupListWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>261</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPushButton" name="findGroupButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>424</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>找群</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createGroupButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>424</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>建群</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="minButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>0</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="changePhotoButton">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>0</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="photoButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>minButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>5</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
